src/Lykke.Service.Stellar.Api.Services/Transaction/TransactionService.cs

```diff
@@ -195,11 +197,6 @@ private async Task<bool> ProcessDwToHwTransaction(Guid operationId, StellarBase.
                 CreatedAt = DateTime.UtcNow
             };
 
-            // save without state to prevent changing of tx hash
-            await _broadcastRepository.InsertOrReplaceAsync(broadcast);
```

Comment:
Don't we need register DW-HW in broadcastRepository any more?

Reply:
We don't need to do it twice now. In previous implementation I didn't change tx hash (timestamp used), and must save broadcast info without state to save hash to make request idempotent. Now we use always-unique value  as hash and need to save broadcast only once.